genet/schedule_elements.py

```diff
@@ -494,16 +511,18 @@ def info(self):
             self.__class__.__name__, self.id, self.route_short_name, len(self.ordered_stops),
             len(self.trips['trip_id']))
 
-    def plot(self, show=True, save=False, output_dir=''):
-        if self.ordered_stops:
-            return plot.plot_graph(
-                nx.MultiGraph(self.graph()),
-                filename='route_{}_graph'.format(self.id),
-                show=show,
-                save=save,
-                output_dir=output_dir,
-                e_c='#EC7063'
-            )
+    def plot(self, output_dir=''):
+        """
+        Plots the route on kepler map.
+        Ensure all prerequisites are installed https://docs.kepler.gl/docs/keplergl-jupyter#install
+        :param output_dir: output directory for the image, if passed, will save plot to html
+        :return:
+        """
+        m = self.kepler_map()
+        if output_dir:
```

Comment:
This same block of code appears in several places, where the only difference seems to be the file name. Maybe we can refactor this into a little function?